modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/package.json

```diff
@@ -12,7 +12,8 @@
 		"classnames": "2.2.6",
 		"frontend-js-web": "*",
 		"moment": "2.24.0",
-		"path-to-regexp": "3.0.0"
+		"path-to-regexp": "3.0.0",
+		"react-flow-renderer": "^9.1.0"
```

Comment:
Hey @naraandrade, Can we use a fixed version of `react-flow-renderer` instead?

Thx